hawkbit-rest/hawkbit-ddi-api/src/test/java/org/eclipse/hawkbit/ddi/json/model/DdiConfigDataTest.java

```diff
@@ -37,56 +37,52 @@
-                ddiStatus.getResult().getFinished().getName(), "test", "data");
-        assertThat(deserializedDdiConfigData.getId()).isEqualTo(id);
-        assertThat(deserializedDdiConfigData.getTime()).isEqualTo(time);
-        assertThat(deserializedDdiConfigData.getStatus().getExecution()).isEqualTo(DdiStatus.ExecutionStatus.CLOSED);
-        assertThat(deserializedDdiConfigData.getStatus().getResult().getFinished()).isEqualTo(
-                DdiResult.FinalResult.SUCCESS);
+        assertThat(serializedDdiConfigData).contains("test", "data");
         assertThat(deserializedDdiConfigData.getMode()).isEqualTo(DdiUpdateMode.REPLACE);
 
     }
 
     @Test
-    @Description("Verify the correct deserialization of a model with a additional unknown property")
+    @Description("Verify the correct deserialization of a model with an additional unknown property")
     public void shouldDeserializeObjectWithUnknownProperty() throws IOException {
         // Setup
-        String serializedDdiConfigData = "{\"id\":123,\"time\":\"20190809T121314\","
-                + "\"status\":{\"execution\":\"closed\",\"result\":{\"finished\":\"success\",\"progress\":null},"
-                + "\"details\":[]},\"data\":{\"test\":\"data\"},\"mode\":\"replace\",\"unknownProperty\":\"test\"}";
+        String serializedDdiConfigData = "{\"data\":{\"test\":\"data\"},\"mode\":\"replace\",\"unknownProperty\":\"test\"}";
 
         // Test
         DdiConfigData ddiConfigData = mapper.readValue(serializedDdiConfigData, DdiConfigData.class);
 
-        assertThat(ddiConfigData.getId()).isEqualTo(123);
-        assertThat(ddiConfigData.getTime()).isEqualTo("20190809T121314");
-        assertThat(ddiConfigData.getStatus().getExecution()).isEqualTo(DdiStatus.ExecutionStatus.CLOSED);
-        assertThat(ddiConfigData.getStatus().getResult().getFinished()).isEqualTo(DdiResult.FinalResult.SUCCESS);
         assertThat(ddiConfigData.getMode()).isEqualTo(DdiUpdateMode.REPLACE);
     }
 
     @Test(expected = MismatchedInputException.class)
     @Description("Verify that deserialization fails for known properties with a wrong datatype")
     public void shouldFailForObjectWithWrongDataTypes() throws IOException {
         // Setup
-        String serializedDdiConfigData = "{\"id\":[123],\"time\":\"20190809T121314\","
-                + "\"status\":{\"execution\":\"closed\",\"result\":{\"finished\":\"success\",\"progress\":null},"
-                + "\"details\":[]},\"data\":{\"test\":\"data\"},\"mode\":\"replace\",\"unknownProperty\":\"test\"}";
+        String serializedDdiConfigData = "{\"data\":{\"test\":\"data\"},\"mode\":[\"replace\"],\"unknownProperty\":\"test\"}";
 
         // Test
         mapper.readValue(serializedDdiConfigData, DdiConfigData.class);
     }
+
+    @Test
+    @Description("Verify the correct deserialization of a model with removed unused status property")
+    public void shouldDeserializeObjectWithStatusProperty() throws IOException {
```

Comment:
Add reason/description for this test:
properties were there before. But are gone now.